docs/conf.py

```diff
@@ -21,6 +21,39 @@
 
 import more_itertools
 
+# -- Preprocess README.rst -----------------------------------------------------
+
+import re
+
+with open('../README.rst', 'r+') as source:
+    readme_file = source.readlines()
+
+# Make sure build directory exists. This is required for github workflow.
+build_dir = '_build'
+if not os.path.exists(build_dir):
+    os.makedirs(build_dir)
```

Comment:
Can this be `os.makedirs(build_dir, exist_ok=True)`?

Reply:
yup